tests/common_docker.py

```diff
@@ -47,8 +47,8 @@ def docker_compose_cmd(arg_list, use_common_files=True):
                                               arg_list)
 
         logging.info("running with: %s" % cmd)
-        subprocess.Popen(cmd, shell=True).wait()
-
+        ret = subprocess.Popen(cmd, shell=True).wait()
+        assert ret == 0, "docker-compose failed, command returned exit code: " + str(ret)
```

Comment:
I want to use this, so I can clearly indicate what failed, CalledProcessError only returns the exit code

> Run command with arguments. Wait for command to complete. If the return code was zero then return, otherwise raise CalledProcessError. The CalledProcessError object will have the return code in the returncode attribute.